src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.EasyRequest.cs

```diff
@@ -370,6 +370,11 @@ internal void SetPossibleRedirectForLocationHeader(string location)
                     {
                         SetCookieOption(newUri);
                     }
+
+                    if (newUri.Scheme == Uri.UriSchemeHttp && _requestMessage.RequestUri.Scheme == Uri.UriSchemeHttps)
+                    {
+                        EventSourceTrace($"Insecure https to http redirect: {0}", (_requestMessage.RequestUri, newUri));
```

Comment:
Remove the interpolation ```$```

Reply:
I missed that you made this comment! I'll try to address that.